content/90.back-matter.md

```diff
@@ -2,3 +2,116 @@
 
 <!-- Explicitly insert bibliography here -->
 <div id="refs"></div>
+
+<style> span.gene_color { color:#02b3e4 } span.disease_color { color:#875442 } span.compound_color { color:#e91e63 } </style> 
+## Supplemental Figures {.page_break_before}
+
+### Calibrated Confidence Scores for Each Edge Type 
```

Comment:
```suggestion
### Confidence Scores for Each Edge Type 
```
What about adding a column for the uncalibrated scores too, since it's not clear that the calibration helped much?